### PR TITLE
Rename TransportTypes to HttpTransportTypes

### PR DESCRIPTION
**Pull request by mikaelm12**
Avoiding naming conflicts by renaming our TransportType enums to HttpTransportTypes.
Issue: https://github.com/aspnet/SignalR/issues/1807

### CODE REVIEW

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Helpers.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 
-using HttpTransportType = Microsoft.AspNetCore.Http.Connections.TransportType;
+using HttpTransportType = Microsoft.AspNetCore.Http.Connections.HttpTransportType;
```
Comment (JamesNK) [inline test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Helpers.cs]: Still needed?

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Negotiate.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@
 using Newtonsoft.Json;
 using Xunit;
 
-using TransportType = Microsoft.AspNetCore.Http.Connections.TransportType;
+using HttpTransportType = Microsoft.AspNetCore.Http.Connections.HttpTransportType;
```
Comment (JamesNK) [inline test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Negotiate.cs]: ^

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Transport.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@
 using Microsoft.AspNetCore.Http.Connections.Client;
 using Xunit;
 
-using HttpTransportType = Microsoft.AspNetCore.Http.Connections.TransportType;
+using HttpTransportType = Microsoft.AspNetCore.Http.Connections.HttpTransportType;
```
Comment (JamesNK) [inline test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Transport.cs]: ^

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs [hunk 4]
```diff
@@ -16,7 +16,7 @@
 using Xunit.Abstractions;
 
 // This is needed because there's a System.Net.TransportType in net461 (it's internal in netcoreapp).
-using TransportType = Microsoft.AspNetCore.Http.Connections.TransportType;
+using HttpTransportType = Microsoft.AspNetCore.Http.Connections.HttpTransportType;
```
Comment (JamesNK) [inline test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs]: etc

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionBuilderExtensionsTests.cs [hunk 5]
```diff
@@ -11,7 +11,7 @@
 using Microsoft.Extensions.Logging.Console;
 using Moq;
 using Xunit;
-using TransportType = Microsoft.AspNetCore.Http.Connections.TransportType;
+using HttpTransportType = Microsoft.AspNetCore.Http.Connections.HttpTransportType;
```
Comment (anurse) [inline test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionBuilderExtensionsTests.cs]: Not needed

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/ResponseUtils.cs [hunk 6]
```diff
@@ -8,7 +8,7 @@
 using Microsoft.AspNetCore.Connections;
 using Newtonsoft.Json;
 
-using TransportType = Microsoft.AspNetCore.Http.Connections.TransportType;
+using HttpTransportType = Microsoft.AspNetCore.Http.Connections.HttpTransportType;
```
Comment (anurse) [inline test/Microsoft.AspNetCore.SignalR.Client.Tests/ResponseUtils.cs]: Not needed

File: test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Helpers.cs (hunk 1 above)
Comment (mikaelm12) [reply test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Helpers.cs]: Nope!

**Pull request merged**